proto/wg/cosmo/platform/v1/platform.proto

```diff
@@ -1187,6 +1191,85 @@ message ForceCheckSuccessResponse {
   Response response = 1;
 }
 
+message OverrideChange {
+  string changeType = 1;
+  optional string path = 2;
+}
+
+message CreateOperationOverridesRequest {
+  string graph_name = 1;
+  string namespace = 2;
+  string operation_hash = 3;
+  string operation_name = 4;
+  repeated OverrideChange changes = 5;
+}
+
+message CreateOperationOverridesResponse {
+  Response response = 1;
+}
+
+message CreateOperationIgnoreAllOverrideRequest {
+  string graph_name = 1;
+  string namespace = 2;
+  string operation_hash = 3;
+  string operation_name = 4;
+}
+
+message CreateOperationIgnoreAllOverrideResponse {
+  Response response = 1;
+}
+
+message RemoveOperationOverridesRequest {
+  string graph_name = 1;
+  string namespace = 2;
+  string operation_hash = 3;
+  repeated OverrideChange changes = 4;
+}
+
+message RemoveOperationOverridesResponse {
+  Response response = 1;
+}
+
+message RemoveOperationIgnoreAllOverrideRequest {
+  string graph_name = 1;
+  string namespace = 2;
+  string operation_hash = 3;
+}
+
+message RemoveOperationIgnoreAllOverrideResponse {
+  Response response = 1;
+}
+
+message GetOperationOverridesRequest {
+  string graph_name = 1;
+  string namespace = 2;
+  string operation_hash = 3;
+}
+
+message GetOperationOverridesResponse {
+  Response response = 1;
+  repeated OverrideChange changes = 2;
+  bool ignore_all = 3;
+}
+
+message GetAllOverridesRequest {
+  string graph_name = 1;
```

Comment:
Lets stay camelCase everywhere.

Reply:
The convention is snake case. That said nobody is following it. Maybe a refactor at some point across the whole file.

Reply:
ok